apps/web/src/pages/Applications/MyApplicationsPage/components/ApplicationCard/ApplicationActions.tsx

```diff
@@ -92,11 +159,12 @@ const DeleteAction = ({ show, application, onDelete }: DeleteActionProps) => {
   return (
```

Comment:
same for archive action

![image](https://user-images.githubusercontent.com/40485260/219791617-69d8181b-b239-4b13-b5b8-e9f9fc9154d2.png)

Reply:
Also fixed in [fix issue with nested p tags in application dialogs](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5744/commits/42f9289e919319fd6e0623f98b52f3802edfafec)